sdk/arm/msi/2018-11-30/armmsi/zz_generated_connection.go

```diff
@@ -7,4 +7,4 @@
 
 package armmsi
 
-const telemetryInfo = "azsdk-go-armmsi/v0.2.0"
+const telemetryInfo = "azsdk-go-armmsi/v0.3.0"
```

Comment:
I think this can remain at `v0.1.0` since we haven't released any modules for this.

Also, I just realized that with our one-module-per-RP, we'll need to include the API version in the telemetry string.

Reply:
Ah, you are seeing the version number bumping because I am releasing this module in my forked and replace it in terraform for my testing. I will make a new PR for the actual release.

I suppose if we need to include api-version here, we might need to do it in autorest.go?

Reply:
Unlikely as the code generator doesn't have this info.  It will likely need to be added when the module version is added.